src/Bicep.LangServer/Program.cs

```diff
@@ -13,6 +16,11 @@ public class Program
         public static async Task Main(string[] args)
             => await RunWithCancellationAsync(async cancellationToken =>
             {
+                string profilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), LanguageConstants.LanguageId);
+                Directory.CreateDirectory(profilePath);
+                ProfileOptimization.SetProfileRoot(profilePath);
```

Comment:
>SetProfileRoot [](http://example.com/codeflow?start=36&length=14)

Are these accessible in netstandard 2.1 projects. If yes, could you move to Core and reuse? If not available in netstandard2.1, let's leave as-is.

Reply:
The `ProfileOptimization` is not available in .net standard, so I extracted only the path part.

Reply:
Makes sense.